src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int figureCount = 4;
+    public static final int limit = 100;
+    public static final int defoultRadius = 10;
+    public static final String defoultColor = "white";
```

Comment:
constants should be `private static final` 
also they should have names in uppercase

Reply:
Why names should be in uppercase?